lib/bolt/node/errors.rb

```diff
@@ -7,7 +7,8 @@ class Node
     class BaseError < Bolt::Error
       attr_reader :issue_code
 
-      def initialize(message, issue_code)
+      # TODO: can we just drop issue code here?
```

Comment:
+1

Reply:
Might technically be a breaking change, `fail_plan` accepts an optional `issue_code`.

Reply:
Remove the todo if we're keeping this?